spec/index.html

```diff
@@ -595,12 +599,12 @@ <h3>Examples</h3>
           </tr></thead>
           <tbody>
             <tr>
-              <td>`_:e0`</td>
-              <td>`:p :q _:e0 .`<br/>`_:e0 :s :u .`</td>
+              <td>`e0`</td>
+              <td>`:p :q e0 .`<br/>`e0 :s :u .`</td>
```

Comment:
As this looks like a serialization of the quad, blank nodes should either be represented using the `_:` prefix, or they should be otherwise distinguished, possibly using `<var>`, `<em>` or similar, which may require some explanation of nomenclature. Easiest is probably to stick with `_:e0` when presenting a quad unless you think it's key that we stick with the abstract syntax, in which case we probably need to define our notation.

Reply:
I must admit I hesitated on this. The problem that I can see is that in the explanation text the bare id is used, as it should be, hence my choice of removing the infamous `_:`. 

Actually, it is simple to distinguish the blank nodes in all those places: they are the only symbols without the `:` character at the beginning.

Reply:
In the updated diagrams, the identifiers are italicized, so maybe that would help. I can do that in an update.